dbot.py

```diff
@@ -202,33 +215,15 @@ def __init__(self):
             return await ctx.send(f"Error: A number is too large for me to store.")
 
     # send error to sentry.io
-    bot.log_exception(error)
+    if isinstance(error, CommandInvokeError):
+        bot.log_exception(error.original, ctx)
```

Comment:
A `CommandInvokeError` is a wrapper from discord.py for exceptions that are raised from inside a command. Logging the original error removes fluff and makes the logged exception title more accurate to what actually happened.